src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 101;
+    private final Random randomBallNumber = new Random();
+    private final ColorSupplier randomBallColor = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(randomBallColor.getRandomColor(),
+                randomBallNumber.nextInt(MAX_BALL_NUMBER));

```

Comment:
method implementation should look like
```
 public Ball getRandomBall() {
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(FIGURE_COUNT));
    }
```
Do not forget add the `ColorSupplier colorSupplier` variable as class field.